File: src/main/java/com/kata/board/entity/Comment.java
package com.kata.board.entity;

import jakarta.persistence.*;

import java.time.LocalDateTime;

@Entity
public class Comment {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String reply;
    @Column(name = "created_date")
    private LocalDateTime createdDate;
    @Column(name = "modified_date")
    private LocalDateTime modifiedDate;
    private Long postId;
    @ManyToOne(fetch = FetchType.LAZY)
    private User user;
}

File: src/main/kotlin/com/kata/board/entity/Comment.kt
package com.kata.board.entity

import jakarta.persistence.*
import org.hibernate.annotations.Comment

@Entity
class Comment constructor(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private val id: Long,

    @Column(columnDefinition = "TEXT")
    @Comment("댓글")
    private var reply: String,

    @ManyToOne(fetch = FetchType.LAZY)
    private val post: Post,

    @ManyToOne(fetch = FetchType.LAZY)
    private val user: User
) : BaseEntity()